api/models.py

```diff
@@ -30,6 +31,14 @@ def validate(cls, value):
         return ObjectId(value)
 
 
+class StatusValues(enum.Enum):
+    """Enumeration to declare values to be used for Node.status"""
+
+    PENDING = "pending"
+    TRUE = "true"
+    FALSE = "false"
```

Comment:
We should have `PASS` and `FAIL` rather than `TRUE` and `FALSE` which were due to the boolean type. They really mean whether the test passed or failed.

Reply:
Updated the PR for this. Need to update README.md after this gets approved.